.circleci/config.yml

```diff
@@ -32,16 +32,19 @@ save_dep: &save_dep
     key: v6-dependency-cache-{{ checksum "yarn.lock" }}
     paths:
       - node_modules
+      - packages/benchmarking/node_modules
```

Comment:
It is unfortunate that we are unable to use globs: https://circleci.com/docs/2.0/caching/#basic-example-of-dependency-caching